packages/web3-net/src/index.ts

```diff
@@ -0,0 +1,20 @@
+import { Web3Context } from 'web3-core';
+
+import * as rpcMethodsWrappers from './rpc_method_wrappers';
+import { NetRPCApi } from './types';
+
+export class Web3Net extends Web3Context<NetRPCApi> {
+	public async getId() {
+		return rpcMethodsWrappers.getId(this);
+	}
+
+	public async isListening() {
+		return rpcMethodsWrappers.isListening(this);
+	}
+
+	public async getPeerCount() {
```

Comment:
Needs to include `returnType` functionality [e.g.](https://github.com/ChainSafe/web3.js/blob/ea41c556130558b37305767a39f25b4306dc0e50/packages/web3-eth/src/rpc_method_wrappers.ts#L43)